src/embed.ts

```diff
@@ -76,6 +76,7 @@ const SVG_CIRCLES = `
 </svg>`;
 
 const I18N = {
+  CLICK_TO_OPEN_ACTIONS: 'Click to view actions',
```

Comment:
Let's make the key and value consistent in terms of open/view.
```suggestion
  CLICK_TO_VIEW_ACTIONS: 'Click to view actions',
```